devlab/tests/config.py

```diff
@@ -22,9 +22,13 @@
         [{'name': 'tenant1', 'role': '_member_'}]}
 ]
 
+user_tenant_roles = [
+    {'user1': [{'tenant': 'tenant1', 'role': 'SomeRole'}]}
+]
+
 # Roles to create/delete
 roles = [
-    {'name': 'SomeRole'}
+    {'name': 'SomeRole'}, {'name': 'SecondRole'}
```

Comment:
'SomeRole' is created on dst cloud before migration (this role won't migrate from src) (SAIC-522). SecondRole is used for role migration testing.